charts/plugin-site/templates/secret.yaml

```diff
@@ -7,10 +7,10 @@ metadata:
 {{ include "plugin-site.labels" . | indent 4 }}
 type: Opaque
 data:
-  github_app_id: {{ required "github.appId is required" .Values.github.appId | toString | b64enc }}
-  github_app_key: {{ required "github.appPrivateKey is required" .Values.github.appPrivateKey | toString | b64enc }}
-  jira_username: {{ required "jira.username is required" .Values.jira.username | b64enc }}
-  jira_password: {{ required "jira.password is required" .Values.jira.password | b64enc }}
+  github_app_id: {{ required "github.appId is required" .Values.github.appId | default "" | toString | b64enc }}
+  github_app_key: {{ required "github.appPrivateKey is required" .Values.github.appPrivateKey | default "" | toString | b64enc }}
+  jira_username: {{ required "jira.username is required" .Values.jira.username | default "" | b64enc }}
+  jira_password: {{ required "jira.password is required" .Values.jira.password | default "" | b64enc }}
```

Comment:
This change is not coherent: either we want to fail if the value is empty (current case with default empty value) or we should remove the `required` stance. The case of defaulting to empty will never be met.

=> you should check, as indicated in https://github.com/jenkins-infra/helm-charts/issues/660, other charts on this repository such as `accountapp` as example.